pkg/3scale/amp/operator/system_searchd_options_provider.go

```diff
@@ -80,6 +80,14 @@ func (s *SystemSearchdOptionsProvider) podTemplateLabels() map[string]string {
 		labels[k] = v
 	}
 
+	if s.apimanager.Spec.System != nil &&
```

Comment:
this `if` is not needed

Reply:
Addressed in this [commit](https://github.com/3scale/3scale-operator/pull/835/commits/f903bf41523372ad2297594e4efac295a12c0665)